tutorials/scripting/cross_language_scripting.rst

```diff
@@ -171,6 +171,30 @@ If that's impossible, you'll see the following error: ``Invalid call. Nonexisten
     my_csharp_node.PrintArray(["a", "b", "c"]) # a, b, c
     my_csharp_node.PrintArray([1, 2, 3]) # 1, 2, 3
 
+Receiving C# collections from GDScript
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example:
+
+.. code-block:: csharp
+
+    // These methods will not be visible to GDScript, and Godot will throw the "Invalid call" error
```

Comment:
I believe that comments should be brought outside into body text if possible, as comments don't currently get translated? See discussion [here](https://github.com/godotengine/godot-docs/issues/4188).

Reply:
Thanks for the suggestions! Made all of the changes. 

I thought this apt to put into the docs as I think i've been burned by this fact for way too long and didn't realize what exactly was happening until now.